smpe/bld/service/ptf-bucket.txt

```diff
@@ -5,5 +5,5 @@
 #UO01948 UO01949 - IO27918 IO27919 - Fri Jul 10 13:23:34 UTC 2020 - Zowe 1.13.0
 #UO01951 UO01952 - IO27949 IO27950 - Wed Aug 12 14:01:06 UTC 2020 - Zowe 1.14.0
 #UO01953 UO01954 - IO27951 IO27952 - Fri Sep 11 12:48:24 UTC 2020 - Zowe 1.15.0
-UO01955 UO01956 - IO27953 IO27954 - 
+UO01955 UO01956 - IO27953 IO27954
 #UO01958 UO01959 - IO27974 IO27975 -
```

Comment:
can you remove the trailing dash of the 1.17 line as I think it might confuse the automated promote.